test/source/mock/fes/fes-constants.ts

```diff
@@ -142,6 +143,16 @@ export const processMessageFromUser3 = async (body: string, fesUrl: string) => {
   });
   expect(decrypted.success).to.equal(true);
   const decryptedMimeMsg = decrypted.content?.toUtfStr();
+  const cryptupDataRegex = /cryptup-data="([^"]*)"/;
+  const cryptupDataMatch = decryptedMimeMsg?.match(cryptupDataRegex);
+  expect(cryptupDataMatch).to.not.be.undefined;
+  expect(cryptupDataMatch && cryptupDataMatch.length).to.be.greaterThan(1);
+  if (cryptupDataMatch) {
+    const cryptupData = base64decode(cryptupDataMatch[1]);
+    const jsonObject = JSON.parse(cryptupData);
```

Comment:
let's remove `if (cryptupDataMatch) {` and use `base64decode(cryptupDataMatch![1])` instead.

it's ok to use `!` in tests, as having `undefined` value will cause test to fail and it'll need to be fixed. while using `!` in extension code can cause some unexpected failures, which won't be handled.

Reply:
Thank you - I think I get the point. It's sensible to use `!` since the value I'm testing with is guaranteed to have value, unlike in extension, which is prone to unexpected error.

Reply:
Exactly👍